contracts/src/core/interfaces/IEigenDADirectory.sol

```diff
@@ -44,94 +44,94 @@ interface IEigenDAAddressDirectory {
 
 /// @title IEigenDAConfigRegistry
 /// @notice Interface for a configuration registry that allows adding and retrieving configuration entries by name.
-///         Supports both bytes32 and bytes types for configuration values, and maintains a checkpointed structure for each configuration entry
+///         Supports bytes types for configuration values, and maintains a checkpointed structure for each configuration entry
 ///         by an arbitrary activation key.
 interface IEigenDAConfigRegistry {
-    /// @notice Adds a 32 byte configuration value to the configuration registry.
+    /// @notice Adds a variable length byte configuration value to the configuration registry using block number as activation key.
     /// @param name The name of the configuration entry.
     /// @param activationKey The activation key for the configuration entry.
     ///                      This is an arbitrary key defined by the caller to indicate when the configuration should become active.
-    /// @param value The 32 byte configuration value.
+    /// @param value The variable length byte configuration value.
     /// @dev The activationKey must be strictly greater than the last activationKey for the same name.
-    function addConfigBytes32(string memory name, uint256 activationKey, bytes32 value) external;
+    function addConfigBlockNumber(string memory name, uint256 activationKey, bytes memory value) external;
 
-    /// @notice Adds a variable length byte configuration value to the configuration registry.
+    /// @notice Adds a variable length byte configuration value to the configuration registry using timestamp as activation key.
     /// @param name The name of the configuration entry.
     /// @param activationKey The activation key for the configuration entry.
     ///                      This is an arbitrary key defined by the caller to indicate when the configuration should become active.
```

Comment:
Some of the docs still mention `arbitrary key`, which is no longer the case.

Consider doing a pass over these docs, to strip out stale language like this, that's has smells from the old design

Reply:
[109580e](https://github.com/Layr-Labs/eigenda/pull/2338/commits/109580e4a0486b63a45b6a6bdaa62f1449be04ef)